client/src/Utils/Timeout/Timeout.ts

```diff
@@ -0,0 +1,3 @@
+export const timeout = (ms: number) => { 
+    return new Promise(resolve => setTimeout(resolve, ms));
```

Comment:
Why turning timeout to promise and not using the regular setTimeout with callback